### PR TITLE
Correct batch signature for scalar  f in or out

### PR DESCRIPTION
**Pull request by ArnoStrouwen**

### CODE REVIEW

File: src/problems/basic_problems.jl [hunk 1]
```diff
@@ -310,27 +310,35 @@ Integral problems are multi-dimensional integrals defined as:
...
 whose geometry matches the space being integrated.
+This space is bounded by the lowerbound `lb` and upperbound `ub`,
+which are `Number`s or `AbstractVector`s with the same geometry as `u`.
 
 ## Problem Type
 
 ### Constructors
 
-IntegralProblem{iip}(f,lb,ub,p=NullParameters();
+IntegralProblem(f,lb,ub,p=NullParameters();
```
Comment (ChrisRackauckas) [inline src/problems/basic_problems.jl:322]: why this change?

Comment (ChrisRackauckas) [inline src/problems/basic_problems.jl:322]:
```suggestion
IntegralProblem{iip}(f,lb,ub,p=NullParameters();
```

Comment (ArnoStrouwen) [reply src/problems/basic_problems.jl]: Well there was already a sentence:
> Additionally, we can supply iip like IntegralProblem{iip}(...) as true or false to declare at
> compile time whether the integrator function is in-place.
